src/lib_archives.mli

```diff
@@ -0,0 +1,13 @@
+open Stdune
+
+type t =
```

Comment:
I'm not against concrete types in general, but since this is a new one: would it make sense to start it abstract, based on how it is used?

Reply:
I usually make the type abstract when the need arises, there's no harm in doing it here. I've done so.